### DISCUSSION TITLE
Hide or change the "autohide" speed of the Android Virtual Gamepad.

### DISCUSSION THREAD

**Discussion by Marcew44** [flyinghead/flycast#1572] [Ideas]
I haven't managed to find the option (nor a discussion or open issue) so i assume it doesn't exist but i basically want to be able to hide the virtual gamepad or at least change how fast it "autohides", currently i think it takes arounds 10 seconds after you touch the screen for it to hide automatically. 
I've been looking around in the code too to see if i could find how to change this but still no luck so even a hint on what file handles the Virtual Gamepad behavior would be appreciated. Thanks in advance.

**Comment by flyinghead** [comment]
This is not configurable at the moment. The 10 sec timeout is set here:
https://github.com/flyinghead/flycast/blob/c6f50fad9d4bb735f02444fddb14f53cfb6bacfc/shell/android-studio/flycast/src/main/java/com/reicast/emulator/emu/VirtualJoystickDelegate.java#L182
